src/stream/stream.service.js

```diff
@@ -1,3 +1,5 @@
+/* global moment */
```

Comment:
I see that, but you don't need this workaround if instead you directly inject it in the factory.